blueman/gui/manager/ManagerDeviceList.py

```diff
@@ -275,13 +275,12 @@ def on_finished(fader: TreeRowFade) -> None:
         klass = get_minor_class(device['Class'])
         # Bluetooth >= 4 devices use Appearance property
         appearance = device["Appearance"]
-        if klass != "uncategorized" and klass != "unknown":
-            # get translated version
-            description = get_minor_class(device['Class'], True).capitalize()
-        elif klass == "unknown" and appearance:
+        if klass != _("Uncategorized") and klass != _("Unknown"):
+            description = get_minor_class(device['Class'])
```

Comment:
```suggestion
            description = klass
```

Reply:
Wondering if you need to approve again, let's try to commit your suggestion :-)

Reply:
Of course. :smile: